steps/http/session.go

```diff
@@ -243,3 +246,16 @@ func (s *Session) StoreResponseHeaderInContext(ctx context.Context, header strin
 	golium.GetContext(ctx).Put(ctxtKey, h)
 	return nil
 }
+
+// ValidateResponseBodyJSONPropertyLength compares the number of elements nested to a JSON property from the HTTP response body with an expected value.
+func (s *Session) ValidateResponseBodyJSONPropertyLength(ctx context.Context, key string, expectedLength string) error {
```

Comment:
Have you tried with: `xxx.#`. According to [documentation](https://github.com/tidwall/gjson), it provides the number of elements in the array. I think it's better instead of introducing a new step.

Reply:
You're right